app.py

```diff
@@ -30,3 +30,31 @@ def get_pcd(filename: str) -> Response:
             os.remove(download_file_path)
         except FileNotFoundError:
             pass
+
+
+@bp.route("/", methods=["GET"])
+def default() -> Any: # TODO fix
+
+    print("APP")
+    return render_template("index.html")
+
```

Comment:
This file was created to only handle serving PCD, image and height map files to/from the front end, hence the flask "files" blueprint. 

Can we move the serving of the index.html to a different file and create a new blueprint for handling the routing for that?